resdk/tests/unit/test_resolwe.py

```diff
@@ -68,8 +68,9 @@ class TestRegister(unittest.TestCase):
 
     @patch('resdk.resolwe.Resolwe', spec=True)
     def setUp(self, resolwe_mock):  # pylint: disable=arguments-differ
-        self.yaml_file = os.path.normpath(os.path.dirname(__file__) + "./../files/bowtie.yaml")
-        self.bad_yaml_file = os.path.normpath(os.path.dirname(__file__) + "./../files/bowtie_bad.yaml")
+        base_dir = os.path.dirname(__file__)
```

Comment:
I think it would be even better to actually save the common base directory to the `base_dir` variable:
```python
base_dir = os.path.abspath(os.path.dirname(__file__) + '/..')
```

and then just use:
```python
self.yaml_file = os.path.join(base_dir, 'files', 'bowtie.yaml')
self.bad_yaml_file = os.path.join(base_dir, 'files', 'bowtie_bad.yaml')
```